flopy/mf6/mfmodel.py

```diff
@@ -1837,12 +1837,12 @@ def load_package(
                 if ftype in self._ftype_num_dict:
                     self._ftype_num_dict[ftype] += 1
                 else:
-                    self._ftype_num_dict[ftype] = 0
+                    self._ftype_num_dict[ftype] = 1
                 if pname is not None:
                     dict_package_name = pname
                 else:
                     dict_package_name = (
-                        f"{ftype}_{self._ftype_num_dict[ftype]}"
+                        f"{ftype}-{self._ftype_num_dict[ftype]}"
```

Comment:
Glad to see this!  I suppose it's possible that it will break some existing workflows, but this is a more sensible default.